server/gitgame/dependency/injection.py

```diff
@@ -22,6 +22,16 @@ def get_github_instance() -> Github:
     return Github(GITHUB_ACCESS_TOKEN)
 
 
+def validate_authors(authors: List[str]) -> List[str]:
```

Comment:
maybe move this into services/

you can add another parameter to this function, so that it is ```def validate_authors(github: Github, authors: List[str]()``` to make the Github instance more injectable